man/docker-top.1.md

```diff
@@ -11,8 +11,9 @@ CONTAINER [ps OPTIONS]
 
 # DESCRIPTION
 
-Display the running process of the container. ps-OPTION can be any of the
- options you would pass to a Linux ps command.
+Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command.
+
+All displayed information are from host's point of view.
```

Comment:
Thank you for the correction. I updated the pull-request accordingly.

On Tue, Dec 1, 2015 at 6:03 PM, moxiegirl <notifications@github.com> wrote:

> In man/docker-top.1.md
> <https://github.com/docker/docker/pull/18240#discussion_r46305423>:
>
> > @@ -11,8 +11,9 @@ CONTAINER [ps OPTIONS]
> >
> >  # DESCRIPTION
> >
> > -Display the running process of the container. ps-OPTION can be any of the
> > - options you would pass to a Linux ps command.
> > +Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command.
> > +
> > +All displayed information are from host's point of view.
>
> @vdemeester <https://github.com/vdemeester> information is --- treated as
> singular in English --- unlike French and German.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18240/files#r46305423>.
>



-- 
Velkoformatovy pigmentovy inkoustovy tisk
Cerveny kopec 2a
639 00 Brno
tel.: +420 721 209 607
e-mail: pospispa@gmail.com
skype: pospispa

Royalty-free photographs:
http://refer.istockphoto.com/ta.php?lc=059498042431004651&atid=43021%7CBannerID%3D43021%7CReferralMethod%3DLink&url=http%3A%2F%2Fwww.istockphoto.com%2Fsearch%2Fportfolio%2F330814%2Fsource%2Fref